ISSUE_TRIAGE.rst

```diff
@@ -0,0 +1,92 @@
+
+This document is a hands-on manual for doing issue and pull request triage for `MLflow issues 
+on GitHub <https://github.com/mlflow/mlflow/issues>`_ . 
+The purpose of triage is to speed up issue management and get community members faster responses. 
+
+Issue and pull request triage has three steps:
```

Comment:
Nit: Maybe 1. 2. 3. is better for labeling steps here

Reply:
To capture offline conversation, we decided to stick with current form as while each step should be done, they don't have to be done in that order.